dataset/src/test/scala/frameless/forward/WriteTests.scala

```diff
@@ -23,4 +41,22 @@ class WriteTests extends TypedDatasetSuite {
     check(forAll(prop[Int] _))
   }
 
-}
\ No newline at end of file
+  test("write parquet") {
+    def prop[A: TypedEncoder](data: List[A]): Prop = {
+      val filePath = s"$TEST_OUTPUT_DIR/${UUID.randomUUID()}"
+      val input = TypedDataset.create(data)
+      input.write.parquet(filePath)
+
+      val dataset = TypedDataset.createUnsafe(
+        implicitly[SparkSession].sqlContext.read.schema(input.schema).parquet(filePath))
```

Comment:
I think just using `sqlContext` should work, no need for `implicitly[SparkSession]`

Reply:
If you can, also change the other test in the same file that does the same.

Reply:
done!